pagopa/io-app
# chore: [IOCOM-1384] FIMS flow error handling/native client

## Short description
added native client usage, along with a temporary error handling solution 

## List of changes proposed in this pull request
- mixpanel tracking (temp)
- native client usage

## How to test
check out fims' dev-server branch, then run a full FIMS flow on both IOS and Android
